articles/architecture-scenarios/spa-api/part-1.md

```diff
@@ -26,13 +26,12 @@ Because SPAs are public clients and cannot securely store a Client Secret since
 
 With this flow, the calling application requests an Access Token over HTTPS with a transformative value&mdash;a Code Verifier (or another type of client secret)&mdash;that can be verified by the authorization server. 
 
-This flow is the most secure for Single-Page Applications.
 
 ## Implicit Flow
 
 The original specifications for OAuth2 introduced the Implicit Flow, a way for SPAs without a backend to obtain Access Tokens and call APIs directly from the browser. However, mitigation strategies are necessary to use the Implicit Flow because tokens are returned in the URL directly from the authorization endpoint as opposed to the token endpoint. Because web technologies have evolved since the introduction of the Implicit Flow, returning an Access Token in a URL provides multiple opportunities for an attack, such as cross-side scripting, token injection, or credential leakage.
 
-With the introduction of the Authorization Code Flow with PKCE, mitigating the risks using the Implicit Flow in SPAs is a thing of the past. While you may still use the Implicit Flow with your SPA, Auth0 recommends using Implicit Flow with traditional server-side applications that can maintain a Client Secret.
```

Comment:
This is more of a hype paragraph and misleading.  Many risks associated with SPAs still exist when using PKCE, but the specific risks of Implicit are mitigated by PKCE.  This statement leads me to think that if I implement PKCE I no longer have risks I need to mitigate.

I would suggest replacing the paragraph with a statement that PKCE is the recommended workflow, and any existing implementation of Implicit should be aware of the risks, and implement the necessary mitigations if they are unable to update to PKCE